Sources/Diagnostics/Networking/DiagnosticsPostOperation.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DiagnosticsPostOperation.swift
+//
+//  Created by Cesar de la Vega on 8/4/24.
+
+import Foundation
+
+typealias DiagnosticsEntries = [String]
```

Comment:
Oh so the whole body of each event is sent as a string then? We should be careful with encoding characters then, but maybe `Body` is already taking care of that? (not sure)

Reply:
I noticed `PostAttributionDataOperation` is doing the same, so I think it is being taken care of, but I will try to find some docs around that

Reply:
So `Body` is `Encodable` and `Body` has the entries, so I think this should work. But I am going to change it to the way `PostAttributionDataOperation` works. It stores the data as `AnyEncodable` instead of `[String:Any]`